scripts/tools/metricExportData.yml

```diff
@@ -0,0 +1,159 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool Experimental Extension for Exporting Metrics Data"
+version: "1.6"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Exporting Metrics Data Experimental Extension Name"
+version: "1.6"
+name: $T_EXPORT_METRICS_DATA_EXP_NAME
+value: '"$XT_experimental_metric_export_data"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Exporting Metrics Data Experimental Extension Version(s)"
+version: "1.6"
+name: $x_export_metric_data_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Maximum count of characters in export data element name"
+version: "1.6"
+name: $T_MAX_METRIC_EXPORT_DATA_ELEMENT_NAME_EXP
+value: "256"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Maximum export data element description string size"
+version: "1.6"
+name: $T_MAX_METRIC_EXPORT_DATA_ELEMENT_DESCRIPTION_EXP
+value: "256"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Export Data format"
+version: "1.6"
+class: $tMetric
+name: $t_metric_export_data_format_exp_t
+members:
+    - type: char
+      name: "name[$T_MAX_METRIC_EXPORT_DATA_ELEMENT_NAME_EXP]"
```

Comment:
Thanks for the feedback.
To allow the data format to be strongly typed, an implementation specific header file would be used to describe it.
This would allow the exported data access to be easier to read and quicker to process.
Also this header file could support backward compatibility of data formats and version reporting for maintainability and error reporting